Jenkinsfile

```diff
@@ -8,7 +8,9 @@ node('rhel8'){
             sh 'mkdir quarkus-ls'
         }
         dir ('quarkus-ls') {
+            echo "Checking out quarkus-ls ${params.QUARKUS_LS_TAG}"
             git url: 'https://github.com/redhat-developer/quarkus-ls.git'
+            sh "git checkout ${params.QUARKUS_LS_TAG}"
```

Comment:
Could something like `git url: 'https://github.com/redhat-developer/quarkus-ls.git', branch: ${params.QUARKUS_LS_TAG}` also work ?

Reply:
Also is it worth using ${params.FORK} instead of redhat-developer, or is that just overkill.

Reply:
I don't know if using the `branch` param will work, I just copied this from vscode-microprofile. It might not be a good idea to use `params.FORK` for cloning `quarkus-ls`, since you might want to use vscode-quarkus and quarkus-ls from different github organizations.